installer-build/payload/ai_sperm_analysis/cli_analyzer.py

```
@@ -0,0 +1,196 @@
+        output_results = format_results_for_csharp(results)
+        
+        if args.output == 'json':
+            # إخراج JSON للتكامل مع C#
+            print(json.dumps(output_results, ensure_ascii=False, indent=2))
+        else:
+            # إخراج نصي للوحة التحكم
+            print_console_results(output_results)
+        
+        return 0  # نجاح
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)